patterns/2-structured/praise-participants.md

```diff
@@ -4,59 +4,60 @@ Praise Participants
 
 ## Patlet
 
-After an inner source contribution, it's important to thank the contributor for their time and effort.
-This pattern gives guidance that not only effectively acknowledges the contribution but also engenders further engagement from the contributor and others.
+When you receive an InnerSource contribution, it's important to thank the contributor for their time and effort.
+Extending your gratiutude not only effectively acknowledges the contribution but also engenders further engagement from the contributor and others.
+Praising contributors' positive contributions to your InnerSource project motivates those contributors (and their managers) to continue investing in the effort.
 
 ## Problem
 
-How can we properly express our gratitude to a contributor for their inner source contribution to a project?
-It can be easy to forget to do so or not know the words or medium to use for adequate effect and sincerity.
-Praise and thanks are easy, affordable ways to keep contributors and their managers motivated and excited to continue.
-A pattern in this area makes it easy to do and ensures that the message comes across clearly and sincerely.
+How can we properly express our gratitude to a contributor for their InnerSource contribution to a project?
+We may not know precisely which words or communication channels we should use to express ourselves sincerely.
+Effective praise for participants follows a pattern, and following this pattern helps ensure that our messages of thanks come across clearly and graciously.
+It also makes us less likely to forget to praise our contributors, which is unfortunately too easy to do.
 
 ## Context
 
-* You are the [Trusted Committer](./trusted-committer.md) or maintainer on an inner source project.
+* You are the [Trusted Committer](./trusted-committer.md) or maintainer on an InnerSource project.
 * You value the community of contributors and want to maintain and grow it.
 
 ## Forces
 
-* You are busy, which makes it easy to forget some soft touches like praise and thanks.
+* You are busy, which makes forgetting some soft touches (like praise and thanks) easier.
 * You may not be someone that is comfortable in social situations or good with words.
 * Peer recognition is very important to job satisfaction and career development.
 
 ## Solutions
 
-It feels good to anyone to be recognized by others.
-In a professional setting, increased recognition is also an avenue to increased influence and growth.
-Any time someone gives to your inner source project, recognize them with a sincere and qualified "thank you".
```

Comment:
The "qualified" here meant that an example should be provided. Are we loosing this information with these changes?

Reply:
I see! Thanks for that context. To me (and perhaps to other readers), the word "qualified" here doesn't carry that particular meaning. I read "qualified" here as "limited" or "restricted" or "with exceptions." I don't think this is what the author was trying to say. Your explanation helps. I can re-write this with the new intention in mind.

Reply:
I've re-written the section to include language specifically about highlighting examples. See how that reads to you.

Reply:
As an aside: it is possible that the "qualified" is direct translation from German, which does not work that well in English :)

Reply:
The last additions brings back the idea of provide examples very clearly.
Thank you :)

(maybe I am just using this pattern myself :))